backend/profiles/serializers.py

```
@@ -71,22 +64,16 @@ def update(self, instance, validated_data):
-        for work_data in works_data:
-            work = works.pop(0)
-            work.position = work_data.get('position', work.position)
-            work.company = work_data.get('company', work.company)
-            work.start_date = work_data.get('start_date', work.start_date)
-            work.end_date = work_data.get('end_date', work.end_date)
-            work.location = work_data.get('location', work.location)
-            work.save()
+        if works_data is not None:
+            instance.workexperience.all().delete()
+            for work_data in works_data:
+                WorkExperience.objects.create(employee=instance, **work_data)
```

Ruff (0.13.3)

`WorkExperience` may be undefined, or defined from star imports

(F405)